public/install.php

```diff
@@ -24,7 +24,7 @@
         \'DEFAULT_ACTION\' => \'index\',
 
         \'DB_TYPE\' => "' . $db_type . '",
-        \'DB_HOST\' => \'127.0.0.1\',
```

Comment:
This has been changed so that docker environment can make use of db.
If you're developing in local, you'd only need to add this entry in `/etc/hosts`-
```
127.0.0.1    mysqlhost
```